docs/docs/workers.md

```diff
@@ -69,6 +69,7 @@ In addition to `--burst`, `rq worker` also accepts these arguments:
 * `--date-format`: Datetime format for the worker logs, defaults to `'%H:%M:%S'`
 * `--disable-job-desc-logging`: Turn off job description logging.
 * `--max-jobs`: Maximum number of jobs to execute.
+* `--serializer`: Serialization to use, i.e. Pickle, JSON or other custom serializer
```

Comment:
Can we clarify that this is a path to serializer object? Something like `Path to serializer (e.g  "rq.serializers.DefaultSerializer" or "rq.serializers.JSONSerializer"`

Reply:
Done.